app/views/components/_document-list.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  group_index ||= false
+  track_clicks = "data-module=track-click" if group_index
+  dimension28 = items.count.to_s
+%>
+<ol class="app-c-document-list" <%= track_clicks %>>
+  <% items.each_with_index do |link, link_index| %>
+    <li class="app-c-document-list__item">
+      <h3 class="app-c-document-list__item-title">
+        <%
+          data = {}
+          if group_index
+            data = {
```

Comment:
Double check this tracking is still needed?
Some sort of `track-group-link-clicks` JS module on the containing group?